backend/putio/error.go

```diff
@@ -12,11 +12,13 @@ import (
 	"github.com/rclone/rclone/lib/pacer"
 )
 
-func checkStatusCode(resp *http.Response, expected int) error {
-	if resp.StatusCode != expected {
-		return &statusCodeError{response: resp}
+func checkStatusCode(resp *http.Response, expected []int) error {
+	for _, code := range expected {
```

Comment:
I suggest you make this

    func checkStatusCode(resp *http.Response, expected ...int) error {

Which is a bit of syntax which means allow any number of int parameters this makes `expected` into a `[]int` so nothing else in this function needs to change